tests/fixtures/valid_marketplace/README.md

```
@@ -0,0 +1,18 @@
+# Fixture marketplace
+
+👋 **Welcome!** This is a marketplace of **2 skills in 1 themed plugins**: a minimal
+tree the test suite points the validator at.
+
+| Plugin | Skills | What it is for |
+|---|---|---|
+| **alpha** | 2 | Exercising the validator |
+
+### alpha
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)